paws_collector.js

```diff
@@ -170,8 +170,8 @@ class PawsCollector extends AlAwsCollector {
         return Object.assign(pawsProps, baseProps);
     };
 
-    prepareErrorStatus(errorString, streamName = 'none') {
-        return super.prepareErrorStatus(errorString, streamName, this.pawsCollectorType);
+    prepareErrorStatus(errorString, streamName = 'none', collectorType = this.pawsCollectorType) {
```

Comment:
why do we need 3rd parameter here? Is it actually used in any collector?

Reply:
When we call prepareErrorStatus from al-aws-collector [done function](https://github.com/alertlogic/al-aws-collector-js/pull/62/files#diff-c06f5f0e08865e628912333b6bb7941cf34fcc02b6fc4163a80e350b5bfd4c47R188) it actually call Paws collector(above )method and 3rd parameter is not match so  I am not getting collectorType, So I have updated this method.